.gitignore

```diff
@@ -1,10 +1,3 @@
+/.idea/
 inc/config.inc.php
-.DS_Store
```

Comment:
I can comment all of them:

-/nbproject - ignore NetBeans stuff, let's keep it for user that will work with this IDE
 -/scripts - ignore scripts from https://github.com/poweradmin/poweradmin-scripts that I use from time to time to update TLDs or create release
 -/tests - tests that I used to test current stability of code (https://github.com/poweradmin/poweradmin-tests), leave it
 -/docs - this can be removed
 -/.settings/ - Eclipse? I used it once, but let's leave it
 -/.buildpath - not sure what it is, remove.
 -/.project - Eclipse? leave it.